deploy.sh

```diff
@@ -188,18 +228,24 @@ pollForActiveDeployments
 
 # 5) Poll / Complete
 function deployRevision() {
+    EXTRA_ARGS=''
+    if [ -n "$INPUT_CODEDEPLOY_CONFIG_NAME" ]; then
+        EXTRA_ARGS="--deployment-config-name $INPUT_CODEDEPLOY_CONFIG_NAME"
+    fi
+    
```

Comment:
Lets split this out to a different pr.

Reply:
Yep, my bad, this should definitely be a separate PR.

Reply:
Done 😁